dynamiqs/integrators/core/fixed_step_stochastic_integrator.py

```diff
@@ -179,52 +177,109 @@ def outer_step(carry, key):  # noqa: ANN001, ANN202
 
         return self.result(saved, infos=self.Infos(nsteps))
 
+    @property
+    @abstractmethod
+    def nmeas(self) -> int:
+        pass
+
     @abstractmethod
-    def forward(self, t: Scalar, y: DSMEState, dW: Array) -> DSMEState:
-        # return (rho_{t+dt}, dY_{t+dt})
+    def forward(self, t: Scalar, y: DiffusiveState, dW: Array) -> DiffusiveState:
+        # return (state_{t+dt}, dY_{t+dt})
         pass
 
 
-class DSMESolveEulerMayuramaIntegrator(DSMEFixedStepIntegrator, SolveInterface):
+class DSSEFixedStepIntegrator(DiffusiveSolveIntegrator, DSSEInterface):
+    @property
+    def nmeas(self) -> int:
+        return len(self.Ls)
+
+
+class DSSESolveEulerMayuramaIntegrator(DSSEFixedStepIntegrator):
+    """Integrator solving the diffusive SSE with the Euler-Mayurama method."""
+
+    def forward(self, t: Scalar, y: DiffusiveState, dW: Array) -> DiffusiveState:
+        psi = y.state
+        L, H = self.L(t), self.H(t)
+        I = eye_like(H)
+
+        # === measurement Y
+        # dY = sqrt(eta) <L+Ld> dt + dW
```

Comment:
nit:
```suggestion
        # dY = <L+Ld> dt + dW
```
(eta does not exist in this context I guess)